src/test/resources/outputs/spark/builtins/datetime.sql

```diff
@@ -7,3 +7,61 @@ SELECT `unix_timestamp`(TIMESTAMP_NTZ '2017-01-02 03:04:20.006') - `unix_timesta
 -- Check UTCNOW()
 --#[datetime-23]
 SELECT `convert_timezone`('UTC', `current_timestamp`()) AS `_1` FROM `default`.`T` AS `T`;
+
+-- #[datetime-41]..#[datetime-44] Spark does not support Time types
+
+--#[datetime-45]
+SELECT `date_diff`(`T`.`col_date`, `T`.`col_date`) AS `_1` FROM `default`.`T_ALL_TYPES` AS `T`;
+
+--#[datetime-46]
+SELECT `date_diff`(`T`.`col_timestamp`, CAST(`T`.`col_date` AS TIMESTAMP_NTZ)) AS `_1` FROM `default`.`T_ALL_TYPES` AS `T`;
```

Comment:
Double checking: Have we verified that casting DATE to TIMESTAMP_NTZ work in spark?